src/api/v1/community_businesses/volunteer_logs.ts

```diff
@@ -107,8 +272,7 @@ const routes: Hapi.ServerRoute[] = [
       auth: {
```

Comment:
@eliascodes does this route need the `-sibling` scope? The payload doesn't include userId's so I imagine this route if just for adding your own logs.

Reply:
It's not something I've yet implemented, but I think it was a feature that was supported by the old API

Reply:
cool, lets leave it for now then.